raster-foundry/raster-foundry
# Simplify backsplash color correct

## Overview

This PR:
- removes the `disableAutoCorrect` QP
- simplifies the color correction logic in backsplash

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new SQL strings have tests~
- ~Any new endpoints have scope validation and are included in the integration test csv~

## Testing Instructions

- I pushed a `/test` branch up so staging should reflect the changes in this PR; feel free to push `test/asu/simplify-color-correct` again in case staging is overwritten
- Go to GroundWork staging -- make sure the recent images show up just fine
- Upload the image in question in [here](https://azavea.slack.com/archives/C089LBS6S/p1613585378118400?thread_ts=1613573510.106100&cid=C089LBS6S) in GW and make sure it displays just fine without disabling the auto color correction

Closes https://github.com/azavea/raster-foundry-platform/issues/1202
